css/style.css

```diff
@@ -0,0 +1,806 @@
+/* ============================== General Styles =========================== */
+
+:root {
+  --side-margin: 18px;
+}
+
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  width: 100%;
+  background-color: #eee;
+  font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+/* ============================== Mobile Design =========================== */
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.header-logo {
+  width: 114px;
+  height: 48px;
+  background-image: linear-gradient(-45deg, rgb(206, 87, 18) 20%, purple 80%);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 0;
+  z-index: 51;
+  visibility: visible;
+}
+
+.space-holder {
+  margin-top: 0;
+  height: 2px;
+}
+
+.header-logo img {
+  position: sticky;
+  top: 5px;
+  left: 10px;
+  height: 40px;
+  width: 80px;
+}
+
+.black-header {
+  background-color: black;
+  display: none;
+  justify-content: space-between;
+  align-items: center;
+  color: #ddd;
+  margin-bottom: 0;
+  font-size: 13px;
+  height: 33px;
+  padding-left: 105px;
+  width: 100%;
+}
+
+.black-header-left {
+  display: flex;
+  justify-content: space-around;
+}
+
+.black-header-left ul {
+  display: flex;
+  list-style-type: none;
+  padding-left: 18px;
+}
+
+.black-header-left ul li,
+.black-header-right ul li {
+  margin-right: 7px;
+}
+
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<ul>` element` and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_